### PR TITLE
Add zig version of snake tutorial

### PR DESCRIPTION
**Pull request by desttinghim**
What it says on the tin! I've mostly copy/pasted and changed what was different. I figured it was better to keep things similar than try to change things too much.

### CODE REVIEW

File: site/docs/tutorials/snake/collision-detection.md [hunk 1]
```diff
@@ -115,6 +116,49 @@ Now you're almost done. Only "Game Over" is left to finish this game.
 
 </TabItem>
 
+<TabItem value="language-typescript">
```
Comment (aduros) [inline site/docs/tutorials/snake/collision-detection.md:119]:
```suggestion
<TabItem value="language-zig">
```

**Review by aduros — COMMENTED** [review]
Thanks! This is looking great.

@christopher-kleine can you please take a look?

**Comment by christopher-kleine** [comment]
I'm pretty clueless about Zig. So I can't help much here. But I noticed something else. Not related to Zig though.

File: site/docs/tutorials/snake/collision-detection.md (hunk 1 above)
Comment (desttinghim) [reply site/docs/tutorials/snake/collision-detection.md:119]: Oops, missed one. Thank you!

**Comment by desttinghim** [comment]
@christopher-kleine What was it you noticed?

**Comment by christopher-kleine** [comment]
Simple: In `Setup Your Project` there is a note titled `Prevent Browser from opening`.
The option `--no-browser` is not correct. It should be `--no-open`. Same with the env var `W4_NO_BROWSER`. It's `W4_NO_OPEN` instead.

But that should be my mistake. But I wouldn't mind if you were to fix this. More commits for you. 😆

**Comment by desttinghim** [comment]
Thanks, I got that change committed as well. Might as well get it fixed.

**Comment by aduros** [comment]
Thanks again for this!

**Pull request merged**